_posts/en/newsletters/2025-12-05-newsletter.md

```diff
@@ -0,0 +1,161 @@
+Bitcoin's consensus rules._
+
+- **LNHANCE soft fork**: moonsettler [proposes][ms ml lnhance] a soft fork for
+  LNHANCE now that all four of its constituent opcodes have updated BIPs and
+  reference implementations. [BIP119][] ([OP_CHECKTEMPLATEVERIFY][topic op_checktemplateverify]),
+  [BIP348][] ([OP_CHECKSIGFROMSTACK][topic op_checksigfromstack]), [BIP349][] (INTERNALKEY), and [BIPs #1699][]
+  (PAIRCOMMIT) add re-bindable signatures and multi-commitments to
+  [tapscript][topic tapscript], and next transaction commitments to all Script
+  versions. The similar tapscript-only opcode package [proposed][gs ml thikcs]
+  by Greg Sanders, et al., has similar capabilities, but lacks general
+  multi-commitments and, being newer, has not yet received as much review on
+  the bulk of the code.
+
+- **Benchmarking The Varops Budget**: Julian [posted][j ml varops] a call to
+  action to benchmark Bitcoin script execution under the [varops budget][bip
+  varops]. The Script Restoration (see [Newsletter #374][news374 gsr]) team asks that users run their
+  [benchmark][j gh bench] and share the results from a wide variety of hardware
+  and operating systems to confirm or improve the chosen varops parameters. In
+  response to Russell O'Connor, Julian also clarified that the varops budget
+  would be used instead of (not in addition to) the sigops budget in the new
+  [tapscript][topic tapscript] version.
+
+- **SLH-DSA (SPHINCS) post-quantum signature optimizations**: On the backdrop of
+  the active discussion around hardening bitcoin against [quantum computing][topic quantum resistance],
+  conduition [presented][c ml sphincs] his work on optimizing the SPHINCS
+  signing algorithm. These optimizations require several megabytes of RAM and
+  locally compiled shaders (highly optimized, cpu-specific machine code — either durably cached or computed on startup). Where
+  applicable, the optimized SPHINCS signing and key generation operations are an
+  order of magnitude faster than the previous state of the art and only two orders
+  of magnitude slower than elliptic curve operations. More importantly, the
+  optimized signature verification is approximately as fast as elliptic curve
+  signature verification.
+
+## Releases and release candidates
+
+_New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates._
+
+- [Core Lightning v25.12][] is a release of this major LN implementation that
+  adds [BIP39][] mnemonic seed phrases as the new default backup method,
+  improves pathfinding, adds experimental [JIT channels][topic jit channels]
+  support, and many other features and bug fixes. Due to breaking database
+  changes, this release includes a downgrade tool in case something goes wrong
+  (see below for more information).
+
+- [LDK v0.2][] is a major release of this library for building lightning
```

Comment:
Seems weve referred to LDK release without the v historically, so will update to "LDK 0.2"